controllers/apply/form-router-next/joi-extensions/bank-numbers.js

```diff
@@ -1,55 +0,0 @@
-'use strict';
```

Comment:
As mentioned in https://github.com/biglotteryfund/blf-alpha/pull/2198 this was unused